src/components/TodoComponents/TodoList.js

```diff
@@ -1,2 +1,24 @@
 // your components will all go in this `component` directory.
 // feel free to change this component.js into TodoList.js
+
+import React from 'react';
+import Todo from './Todo';
+
+const TodoList = props => {
+    
+    return (
+        <ul>
+            {props.todos.map(todo => {
```

Comment:
Good! Mapping through a property passed down from a parent component's state is the best way to handle this.